src/twisted/python/_setup.py

```diff
@@ -364,10 +364,15 @@ def _checkCPython(sys=sys, platform=platform):
 _isCPython = _checkCPython()
 
 notPortedModules = [
+    "twisted.internet.glib2reactor",
```

Comment:
To make this more actionable: if this is not necessary in this branch, please revert this hunk out.